contrib/babelfishpg_tsql/sql/information_schema_tsql.sql

```diff
@@ -717,3 +717,29 @@ CREATE OR REPLACE VIEW information_schema_tsql.routines AS
 GRANT SELECT ON information_schema_tsql.routines TO PUBLIC;
 
 SELECT set_config('search_path', 'sys, '||current_setting('search_path'), false);
+
+/*
+* CONSTRAINT TABLE USAGE
+*/
+CREATE OR REPLACE VIEW information_schema_tsql.constraint_table_usage
```

Comment:
what release these are part of? this is upgrade script for lama to manatee. If it is for narwhal, we should create a new upgrade script (I am not sure if branch is open for Narwhal dev or not)

Reply:
We should have the view definition before this.
```
SELECT set_config('search_path', 'sys, '||current_setting('search_path'), false);
```